File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/LastDayData.java

package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LastDayData {

    @SerializedName("energy")
    @Expose
    private Double energy;

    public Double getEnergy() {
        return energy;
    }

    public void setEnergy(Double energy) {
        this.energy = energy;
    }

}

File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/LastDayData.kt
package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LastDayData {

    @SerializedName("energy")
    @Expose
    var energy: Double? = null

}
